### PR TITLE
Function updated to pick regions from config file

### PR DESCRIPTION
**Pull request by avasisht**

### CODE REVIEW

File: disable_security_hub.py [hunk 1]
```diff
@@ -10,8 +10,6 @@
 from boto3.session import Session
 
 logging.basicConfig(level=logging.INFO)
-
-
```
Comment (chrismckinnel) [inline disable_security_hub.py]: This breaks my pep8 compliance, can we add these blank lines back in?

File: disable_security_hub.py [hunk 2]
```diff
@@ -120,18 +114,10 @@ def disable_security_hub_in_region(profile_name, region_name):
...
-def get_available_regions(service):
-    session = Session()
-    return session.get_available_regions(service)
-
-
 def get_config():
     with open('config.yml') as yaml_file:
-        config = yaml.load(yaml_file)
+        config = yaml.safe_load(yaml_file)
```
Comment (chrismckinnel) [inline disable_security_hub.py:125]: +1 nice

**Review by chrismckinnel — COMMENTED** [review]
Thanks Abhishek, if you fix those blank lines I'll merge this in.

**Review by avasisht — COMMENTED** [review]
Added the blank lines to the code.

**Review by chrismckinnel — APPROVED** [review]

**Pull request merged**